packages/webapp/public/locales/en/translation.json

```diff
@@ -1995,6 +1995,7 @@
     "SELECT_DATE": "Select the task date",
     "SELECT_TASK_LOCATIONS": "Select the task location(s)",
     "SELECT_WILD_CROP": "This task targets a wild crop",
+    "SOIL_AMENDMENT_LOCATION": "Select a soil amendment location",
```

Comment:
Thank you for fixing this!

I was hoping no more strings, but if we are going to parallel the crop plan UI, do we also need some form of this subtext on the bottom?

![Screenshot 2024-07-17 at 1 19 10 PM](https://github.com/user-attachments/assets/c0ece6c1-eee2-4a77-989e-b1b2301eca7b)

Reply:
Which task is that on? I am looking at irrigation and don't see it.

Reply:
Re read this -- crop plan --- I was not paralleling crop plan. More string changes to come too with buffer zone I think!

Reply:
Oh okay, irrigation task precedent is totally fine with me 👍 I just saw "crop" in your PR description so that's what I assumed you were referring to.

Reply:
Should say "Select the soil amendment location(s)" though rather than changing to singular